woocommerce/woocommerce
# Added notice to check if uploads directoy is public

### All Submissions:






### Changes proposed in this Pull Request:


Check if our uploads directory have some protection.

Closes #26186.

### How to test the changes in this Pull Request:

1. Check out this branch
2. Remove the `index.html` and `.htaccess` file from `wp-content/uploads/woocommerce_uploads`.
3. Run `WC_Admin_Notices::reset_admin_notices()` to reset all messages.
3. Go to the WordPress admin dashboard and check for the new notice.
4. Create a `index.html` inside `wp-content/uploads`, clear the `_woocommerce_upload_directory_status` transients (`delete_transient('_woocommerce_upload_directory_status')`).
5. Check if the notice is gone.

Extra steps to test with an `.htaccess` file:

1. Remove the `index.html` created.
2. Run `WC_Admin_Notices::reset_admin_notices()` again to reset all messages.
3. Create a `.htaccess` file with `deny from all`.
4. Check if the notice is gone.

### Other information:




### Changelog entry

> Enhancement - Added notice about public uploads directory.
